modules/cluster-logging-eventrouter-deploy.adoc

```diff
@@ -107,7 +107,7 @@ objects:
 parameters:
   - name: IMAGE
     displayName: Image
-    value: "registry.redhat.io/openshift-logging/eventrouter-rhel8:latest"
+    value: "registry.redhat.io/openshift-logging/eventrouter-rhel8@sha256:4acc0e5cb837972d812ceb6ce3be5938b121c167f4668ef7b8b626e353296014"
```

Comment:
I suggest use tag 'v0.3'. Currently, we can use registry.redhat.io/openshift-logging/eventrouter-rhel8:v0.3 for all logging version > 5.0

Reply:
Updated to use tag v0.3 and added brief explanation to that section of template - can you please approve? Thank you!